### DISCUSSION TITLE
Who can become a collaborator and how?

### DISCUSSION THREAD

**Discussion by hellonearth311** [Prodigy-Hacking/ProdigyMathGameHacking#720] [(A) Anything]
I want to know, please. @PatheticMustan

**Comment by PatheticMustan** [comment]
We look at people who contribute, and pick from them

**Reply by NX721** [reply]
Lol im a contributor

**Reply by PatheticMustan** [reply]
you also don't know enough

**Reply by hellonearth311** [reply]
You mean you'll look my fork (hellonearth311/ProdigyMathGameHacking) and then choose...
